src/app/messages-page/messages-inbox/messages-inbox.component.ts

```diff
@@ -13,11 +12,25 @@ import { Location } from "@angular/common";
 export class MessagesInboxComponent implements OnInit, OnChanges {
   static CONTACT_US_USERNAME = "clippy";
 
+  static QUERYTOTAB = {
+    "all":        "All",
+    "my-holders": "My Holders",
+    "custom":     "Custom",
```

Comment:
let's define these as static variables as well so we aren't checking strings all over the place.

Reply:
I built the tabs here to match how they work on the creator's profile